docs/getting-started/concepts.md

```diff
@@ -281,7 +281,7 @@ function useFakeFetch(URL) {
       fetch(URL, { signal: controller.signal })
         .then((res) => res.json())
         .then((data) => {
-          if (controller.aborted) return;
+          if (signal.aborted) return;
```

Comment:
```suggestion
          if (controller.signal.aborted) return;
```

This still needs to reference `controller`, as no `signal` variable is established.